docs/install.md

```diff
@@ -72,13 +71,30 @@ NOTE: A TLS Cert and Key need to be generated for the Webhook. MagTape has an in
 
 By default MagTape will handle creation and rotation of the required TLS cert/key automatically. In cases where you need to BYOC, you can adjust the configuration.
 
-### Specify a different secret name
+### Create the secret
 
-Reference the `MAGTAPE_TLS_SECRET` option in the [configuration options](#configuration-options) section.
+Create the secret name called `magtape-tls` with the annotation `magtape-byoc` in the `magtape-system` namespace. The `cert.pem`, `key.pem` must be provided in the data field of the secret. This secret must exist prior to install MagTape.
```

Comment:
suggestion(blocking): I think it would be good to add an imperative example of creating a Kubernetes secret with the appropriate names/keys/etc.

```
$ kubectl create secret
```

Would be great, but not required if you could get it down to a one-liner to also throw in the annotation.

Reply:
thanks for suggestion. I just updated to imperative example of creating k8s secret. 

FYI: one problem I ran into is that, if I create tls secrete using  `kubectl create secret tls magtape-tls --cert=cert.pem --key=key.pem`, the secrete data field will be with `tls.crt`, `tls.key`, however the existing python code is checking the data field `cert.pem` and `key.pem`, so there will be errors in init. 

so I changed to `kubectl create secret generic magtape-tls --from-file=cert.pem=<path/to/cert.pem> --from-file=key.pem=<path/to/key.pem> --namespace magtape-system`. 

I did the test in local, the init container started without errors.  (there is another problem I ran into, which I will comment as bellow in another thread.)

Reply:
Ok, cool, this is perfect. 

Not pertinent to this PR/Issue, bu it does make me wonder if the expected field names should better align with the `kubectl create secret tls ...` syntax. There's a separate request for enhancement on the [imageswap](https://github.com/phenixblue/imageswap-webhook/issues/58) project to allow for customization of those field names/keys (the imageswap project uses the same init-container code as MagTape).